packages/v4/patternfly-docs/content/accessibility/checkbox/checkbox.md

```diff
@@ -3,17 +3,78 @@ id: Checkbox
 section: components
 ---
 
-A **checkbox** is used to select a single item or multiple items, typically to choose elements to perform an action or to reflect a binary setting.
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
 
-Keyboard users will be able to interact with the checkbox component by pressing **Space** to select the input. We have added any attributes necessary to make it properly announced to screen readers.
+## Accessibility
 
-In general, the checkbox component already has accessibility built in. The **standalone input variant** will need special consideration.
+To implement an accessible PatternFly **checkbox** component:
 
-To make the checkbox **standalone input variant** accessible, add an `aria-label` with a label for the input. You can also use `aria-labelledby` to associate the checkbox with visible text on the page to explain the checkbox. 
+- Ensure the checkbox can be navigated to and interacted with via keyboard and other assistive technologies such as a screen reader
+- Ensure the checkbox can be toggled by clicking its visible text label, if it has one
+- Ensure the checkbox label has hover styles to indicate that it can be clicked
+- Ensure the checkbox input provides enough clickable space on its own
+- Provide an `aria-label` if the checkbox does not have a visible text label
+- Place related checkboxes inside of a list element, and that list inside of an element that has the `role="group"` attribute
 
-<br/>
-If you’d like to customize the checkbox component’s accessibility, edit the following:
+For the PatternFly React library:
 
-| React component| React prop | Which HTML element it appears on in markup | Explanation | 
-|---|---|---|---|
-| Checkbox | aria-label | .pf-c-check__input | Aria-label or accessible text for the checkbox. |
\ No newline at end of file
+- Include the `isRequired` prop if the checkbox must be checked
+
+For the HTML/CSS library:
+
+- Include the `required` attribute if the checkbox must be checked
```

Comment:
I think I agree that the required label seems unnecessary in the top section. I think it falls outside of the TLDR of a11y criteria wdyt?

Reply:
Yeah I wasn't sure how often a single, required checkbox would be used but put it there in case. I can definitely remove it, though, and save it for the tables